app/static/js/searchUser.js

```diff
@@ -0,0 +1,31 @@
+export default function searchUser(inputId, ajaxUrl, formId){
+  var query = $(`#${inputId}`).val()
+
+  $(`#${inputId}`).autocomplete({
+    minLength: 2,
+    source: function(request, response){
+
+      $.ajax({
+        url: `/searchUser/${query}`,
+        type: "GET",
+        dataType: "json",
+        success: function(dictToJSON) {
+          response($.map( dictToJSON, function( item ) {
+            return {
+              label: item,
+              value: dictToJSON[item]
+            }
+          }))
+        },
+        error: function(request, status, error) {
+          console.log(status, error);
+        }
+      })
+    },
+     select: function(event , ui) {
+       var user = ui.item.value
+       $(`#${inputId}`).val(ui.item.value)
+       $(`#${formId}`).submit()

```

Comment:
we should check if the element exists before trying to submit it. Do you get a console error right now? Because your searchUser invocation in searchStudents.js doesn't pass a formId.

Reply:
I forgot to remove the ajaxURL parameter. Since all requests will be routed to the same URL, that's no longer needed. I will get rid of it.

Reply:
Since now the only parameters one can pass are the input id and form id, I don't think we need to check for their existence because whoever is using this function would have to pass ids of ones that exist.